libraries/dialogs/src/main/res/layout/fragment_dialog.xml

```diff
@@ -25,8 +26,9 @@
                 android:layout_width="match_parent"
                 android:layout_height="?attr/actionBarSize"
                 android:background="#f5f5f5"
```

Comment:
This color is used in multiple places. The hardcoded value may change with color resources.

Reply:
Done.